From 5527fa53400c558eaf2bf109643f54e38d12e729 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Thu, 6 Aug 2020 12:47:38 -0500
Subject: [PATCH 49/49] arm64: defconfig: Enable ADV7535 Audio

The Audio portion of the HDMI bridge chip isn't enabled by default,
but without it, there won't be HDMI audio.

This patch enables the audio portion to be built.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 arch/arm64/configs/defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index 641085f1b64c..bc8c37dee060 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -496,6 +496,7 @@ CONFIG_DRM_PANEL_SEIKO_43WVF1G=y
 CONFIG_DRM_PANEL_RAYDIUM_RM67191=y
 CONFIG_DRM_NXP_SEIKO_43WVFIG=y
 CONFIG_DRM_I2C_ADV7511=y
+CONFIG_DRM_I2C_ADV7511_AUDIO=y
 CONFIG_DRM_ITE_IT6263=y
 CONFIG_DRM_IMX_LDB=y
 CONFIG_DRM_IMX_NWL_DSI=y
-- 
2.17.1

